helm/.generate/gen-helm-sample.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
It seems possible that we could have more than 1 sample per type, so I suggest we call this `gen-helm-memcached.sh`